core/utils/ether.h

```diff
@@ -25,6 +25,8 @@ struct[[gnu::packed]] Ethernet {
     // Returns "aa:bb:00:11:22:33" (all in lower case)
     std::string ToString() const;
 
+    void CopyFrom(const Address &o);
```

Comment:
I really thing the method name and parameter are self explanatory,  
I usually prefer in such cases to write less then more.
What do you say?